docs/source/tutorials/slowmo_ddp.rst

```diff
@@ -84,18 +50,20 @@ points at which ``use_slowmo`` is used below help demonstrate these changes.
             for (data, target) in dataloader:
                 data, target = data.to(rank), target.to(rank)
                 # Train
-                model.zero_grad()
+                if not use_slowmo:
+                    model.zero_grad()
                 outputs = model(data)
                 loss = loss_fn(outputs, target)
                 loss.backward()
                 optimizer.step()
                 if use_slowmo:
+                    model.zero_grad()
```

Comment:
I assume it is important to have it here for SlowMo, but I don't remember why: it would be good to explain it in the docstring of `perform_slowmo()` and refer to this doc here.

Reply:
Thanks for the update, a couple of follow-up comments on this point:

1. Minor: would it be possible for the doc link to directly point to the `perform_slowmo` part of the page? (no big deal if not possible)

2. How does this save memory? According to the doc (https://pytorch.org/docs/stable/generated/torch.nn.Module.html) it won't flush the tensors unless `set_to_none` is set to True

Reply:
1. Have fixed this. The link is a little ugly but it has very little chance of breaking in the future, so it might be good to go ahead with

2. Ahh nice catch, I've fixed that. In the fairseq repo, setting to None was the default behavior of zero_grad so I got confused about that